### PR TITLE
Add normalization and activation ops to operator benchmarks

### PR DESCRIPTION
**Pull request by jainapurva**
We're adding some more ops to the benchmarking: 

Normalization ops:
- LayerNorm
- RMSNorm
- BatchNorm1d
- BatchNorm2d
- BatchNorm3d
- GroupNorm

Activation ops:
- nn.GELU
- nn.SiLU
- nn.ReLU
- nn.LeakyReLU

### CODE REVIEW

File: .ci/pytorch/test.sh [hunk 1]
```diff
@@ -1805,7 +1805,7 @@ test_operator_microbenchmark() {
   cd "${TEST_DIR}"/benchmarks/operator_benchmark
 
   # NOTE: When adding a new test here, please update README: ../../benchmarks/operator_benchmark/README.md
-  for OP_BENCHMARK_TESTS in matmul mm addmm bmm conv optimizer; do
+  for OP_BENCHMARK_TESTS in activation norm; do
```
Comment (jainapurva) [inline .ci/pytorch/test.sh]: Temporary change, for faster CI runs

**Review by slayton58 — APPROVED** [review]
Looks good! 

I was going to comment that we needed to benchmark the `swish` function for swiglu, but I just learned that swish == SiLU, so all good :)
